cypress/e2e/requests.cy.js

```diff
@@ -0,0 +1,26 @@
+describe('Viewing all requests', () => {
+  it('does not show a request list if the user is logged out.', () => {
+		// Visit a protected route in order to allow cypress to set the cookie and mock the login
+		cy.visit("/requests")
+
+		cy.get('div.alert-heading').contains('Unauthorized').then(() => {
+			cy.log("A logged out user is not able to view requests.")
+		})
+	})
+  
+	it("shows the user's request list if they are logged in.", () => {
+		// Call the custom cypress command to log in
+		cy.login(Cypress.env('TEST_SCIENTIST_USER'), Cypress.env('TEST_SCIENTIST_PW'))
+
+		// Visit a protected route in order to allow cypress to set the cookie and mock the login
+		cy.visit("/requests")
+
+    // check that either the requests are showing, or that the user has no requests
+		const requestListExists = cy.get('article.request-item').should('exist') || 
+    cy.get('p.no-requests').contains('You do not have any requests yet.')
```

Comment:
ref: https://github.com/scientist-softserv/webstore/pull/197#discussion_r1108782043

Reply:
for now. I've unchecked the top box on the ticket under this section (seen below) so we don't lose track that we need to handle this properly.
![image](https://user-images.githubusercontent.com/29032869/219439787-82eb66ea-d73b-4f2b-9dde-67a104bf59af.png)

Reply:
there is a way with cypress fixtures - if we did that and only wanted to use mock data, we would need to mock every single api call- this is an example of that: https://blog.bitsrc.io/mocking-http-calls-in-cypress-end-to-end-tests-fa2e6b7caaf7

it can be done but it isn't really in the scope of this ticket